ios/Classes/QRCaptureView.m

```diff
@@ -82,6 +82,8 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
             [device setTorchMode:AVCaptureTorchModeOff];
         }
         [device unlockForConfiguration];
+    } else if ([call.method isEqualToString:@"dispose"]) {
+        [self dealloc];
```

Comment:
> The runtime will do that for you when the object is no longer needed (see [Clang LLVM ARC document, chapter 7.1.2](http://clang.llvm.org/docs/AutomaticReferenceCounting.html#dealloc)). Also, you should never call dealloc directly! It is meant to be called by Objective-C only when it's time to clean up the object.

Nice to know, thank you! I'm closing this PR since @xuzhongpeng already fix the issue.